bug: make install: cannot stat '../bindings/c/tree-sitter-php.h': No such file or directory

### Did you check existing issues?

- [x] I have read all the [tree-sitter docs](https://tree-sitter.github.io/tree-sitter/using-parsers) if it relates to using the parser
- [x] I have searched the existing issues of tree-sitter-php

### Tree-Sitter CLI Version, if relevant (output of `tree-sitter --version`)

_No response_

### Describe the bug

`make install` says:

```
/usr/bin/make -C php install
make[1]: Entering directory '/builddir/build/BUILD/tree-sitter-php-0.24.0-build/tree-sitter-php-0.24.0/php'
install -d '/builddir/build/BUILD/tree-sitter-php-0.24.0-build/BUILDROOT/usr/include'/tree_sitter '/builddir/build/BUILD/tree-sitter-php-0.24.0-build/BUILDROOT/usr/lib64/pkgconfig' '/builddir/build/BUILD/tree-sitter-php-0.24.0-build/BUILDROOT/usr/lib64'
install -m644 ../bindings/c/tree-sitter-php.h '/builddir/build/BUILD/tree-sitter-php-0.24.0-build/BUILDROOT/usr/include'/tree_sitter/tree-sitter-php.h
install: cannot stat '../bindings/c/tree-sitter-php.h': No such file or directory
make[1]: Leaving directory '/builddir/build/BUILD/tree-sitter-php-0.24.0-build/tree-sitter-php-0.24.0/php'
make[1]: *** [../common/common.mak:72: install] Error 1
make: *** [Makefile:4: install] Error 2
```

It appears that `tree-sitter-php.h` was moved, but the makefiles were not updated to reflect the new location.

### Steps To Reproduce/Bad Parse Tree

See https://kojipkgs.fedoraproject.org//work/tasks/8344/135968344/build.log for a detailed log.

### Expected Behavior/Parse Tree

See https://kojipkgs.fedoraproject.org//packages/tree-sitter-php/0.23.12/2.fc43/data/logs/x86_64/build.log

### Repro

```PHP
make install
```

--- a/common/common.mak
+++ b/common/common.mak
@@ -69,7 +69,7 @@ $(PARSER): $(SRC_DIR)/grammar.json
 
 install: all
 	install -d '$(DESTDIR)$(INCLUDEDIR)'/tree_sitter '$(DESTDIR)$(PCLIBDIR)' '$(DESTDIR)$(LIBDIR)'
-	install -m644 ../bindings/c/$(LANGUAGE_NAME).h '$(DESTDIR)$(INCLUDEDIR)'/tree_sitter/$(LANGUAGE_NAME).h
+	install -m644 ../bindings/c/tree_sitter/$(LANGUAGE_NAME).h '$(DESTDIR)$(INCLUDEDIR)'/tree_sitter/$(LANGUAGE_NAME).h
 	install -m644 $(LANGUAGE_NAME).pc '$(DESTDIR)$(PCLIBDIR)'/$(LANGUAGE_NAME).pc
 	install -m755 lib$(LANGUAGE_NAME).a '$(DESTDIR)$(LIBDIR)'/lib$(LANGUAGE_NAME).a
 	install -m755 lib$(LANGUAGE_NAME).$(SOEXT) '$(DESTDIR)$(LIBDIR)'/lib$(LANGUAGE_NAME).$(SOEXTVER)

